flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": "x86_64"
```

Comment:
this is realy unfortunate as Electron support that platform.

Reply:
```suggestion
  "only-arches": ["x86_64"]
```

that's the proper syntax. It's not a seal of approval though.

Reply:
Well, I don't have the possibility to build it on aarch64, I only have x86_64 HW.

Reply:
there are plenty of electron apps built from source on flathub.